src/components/Ledger/LedgerPane.js

```diff
@@ -32,7 +33,7 @@ class LedgerPane extends Component {
     return (
       <PaneMenu>
         <button type="button" id="clickable-closenewfunddialog" onClick={onCancel} title="close" aria-label="Close New Ledger Dialog">
```

Comment:
Maybe it would be better to use `<IconButton>` here? Like this:
```
<IconButton
  icon="times"
  id="clickable-closenewfunddialog"
  onClick={onCancel}
  title="Close"
  aria-label="Close New Ledger Dialog"
/>
```

Reply:
That's way better approach, I'll make an update now.